### ISSUE TITLE
Xcode 15 beta 8

### ISSUE THREAD

**Issue by leohidalgo** [cgrindel/rules_swift_package_manager#551]
When I try to execute the rule using `Xcode 15 beta 8` I get the following error:
```
INFO: Invocation ID: 1c42f3e5-819a-4fec-a747-cd07b3030255
INFO: Analyzed target //:swift_update_pkgs (0 packages loaded, 0 targets configured).
INFO: Found 1 target...
Target //:swift_update_pkgs up-to-date:
  bazel-bin/swift_update_pkgs-runner.bash
  bazel-bin/swift_update_pkgs
INFO: Elapsed time: 0.228s, Critical Path: 0.00s
INFO: 1 process: 1 internal.
INFO: Build completed successfully, 1 total action
INFO: Running command line: bazel-bin/swift_update_pkgs
gazelle: index 0 expected to be type 'string', but was type 'map[string]interface {}'
make: *** [spm] Error 1
```

I'm using the latest version: `0.8.0`

**Comment by denis15yo** [comment]
Is there a planned fix for this problem in the near future?

**Comment by cgrindel** [comment]
[This PR](https://github.com/cgrindel/rules_swift_package_manager/pull/631) just landed, yesterday. I believe that these are the same issue.

**Comment by cgrindel** [comment]
@AttilaTheFun Do you think this is the issue that you fixed in #631?

**Comment by AttilaTheFun** [comment]
@cgrindel that looks like the same issue. @denis15yo can you try the new version / commit and see if that fixes it?

**Comment by cgrindel** [comment]
The fix is in [0.13.0](https://github.com/cgrindel/rules_swift_package_manager/releases/tag/v0.13.0).
